scipy-1.0/paper.tex

```diff
@@ -1262,6 +1205,10 @@ \section*{Competing Interests}
 %financial interests statement} on behalf of all authors of the paper. This
 %statement must be included in the submitted article file.
 
+\section*{Consortium}
+\subsection*{SciPy Contributors}
```

Comment:
```suggestion
\subsection*{SciPy 1.0 Contributors}
```
I think this may deserve some words on who was offered to be listed here.

Reply:
We say in the author contribution statement what author contributions were. Can you suggest an amendment?